_posts/2016-08-18-On-Emission.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "On our implementation of React Native"
+date: 2016-08-18 12:17
+author: orta
+categories: [tooling, mobile, eigen, node, reactnative]
+series: React Native at Artsy
+---
+
+<center>
+ <img src="/images/emission/emission-logo-artsy.svg" style="height:300px;">
+</center>
+
+I arrived fashionably late to the React Native party in Artsy. I had been a part of our [Auctions Team][auctions_team], where we worked in Swift with [some light-FRP][interstellar]. We were not affected by 4 months of work on moving to React Native, at all. 
```

Comment:
I would emphasise that it happened simultaneously (which I understand from your following paragraph is what you are trying to get at I think.) 

So maybe something like:

"We were not affected by 4 months of work on moving to React Native, that happened simultaneously, at all."